### PR TITLE
More comprehensive documentation of Github gists

### PR DESCRIPTION
**Pull request by wes-at-pieces**
Combined and refactored information from "[Import Gists from Github](https://docs.pieces.app/product-highlights-and-benefits/import-gists-from-github)" and "[One Click Snippet Sharing](https://docs.pieces.app/features/one-click-snippet-sharing)" to produce a more comprehensive single documentation of Github gists, now renamed accordingly from "Import Gists from Github" to "Sharing to and From Github Gists"

- The original "Import Gists from Github" only talked about snippets moving from Gist -> Pieces
- The "One Click Snippet Sharing" only talks about Pieces -> Gist
- The new "Sharing to and From Github Gists" replaces "Import Gists from Github" and talks about both Gist -> Pieces **and** Pieces -> Gist.

### CODE REVIEW

**Comment by jordan-pieces** [comment]
just need to add in the sidebar configuration for your new page  style - i would likely just keep them both there and redirect them to the new page for now and then in a week or so we can get rid of the pages that we don't want all together - but take a look at docs.json on this repo and then also you can find some of the redirect pages - just find a file (like `/one-click-snippet-sharing` that redirects to /features/on-click-snippet-sharing`) and use that as an example.

once the sidebar is repaired along with adding a new section, and the links are g2g you should be fine and we can have georgia go through it when she is back next week @wes-at-pieces

File: docs/product-highlights-and-benefits/sharing-to-and-from-github-gists.mdx [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+title: Sharing To & From GitHub Gists
+description: GitHub Gists are a great way to share code with a group of people for public visibility, and integrate your snippets with your GitHub repositories. Pieces makes it easy to update your Gist repository by enabling you to share saved code snippets within Pieces as Gists, right from the desktop app.
```
Comment (jordan-pieces) [inline docs/product-highlights-and-benefits/sharing-to-and-from-github-gists.mdx:3]: this is what is going to be used as the meta description so we should just make sure that it is below the appropriate length

File: docs/product-highlights-and-benefits/sharing-to-and-from-github-gists.mdx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+---
+
+# Import Gists from GitHub
+
+<Image zoom src="https://storage.googleapis.com/pieces_static_resources/pfd_wiki/IMPORT_GIST.gif" />
```
Comment (jordan-pieces) [inline docs/product-highlights-and-benefits/sharing-to-and-from-github-gists.mdx:8]: just need to get some alt text on here

File: docs/product-highlights-and-benefits/sharing-to-and-from-github-gists.mdx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+Before proceeding to work with Gists from GitHub in Pieces, make sure that you are signed in. _If you are signed in to both Google and GitHub, be sure that the **primary emails on both accounts match**._
+
+## Sharing Pieces snippets as GitHub Gists
+
+GitHub Gists are a great way to share code with a group of people for public visibility, and integrate your snippets with your GitHub repositories. Pieces makes it easy to update your Gist repository by enabling you to share saved code snippets within Pieces as Gists, right from the desktop app.
+
+<Image zoom alt={"Generating a shareable link in the Pieces desktop app"} src="https://storage.googleapis.com/pieces_static_resources/pfd_wiki/SHAREABLE_LINK.gif" />
+
+1. After signing in and ensuring that you are **connected to the cloud**, select any snippet in the desktop app.
```
Comment (jordan-pieces) [inline docs/product-highlights-and-benefits/sharing-to-and-from-github-gists.mdx:18]: would link this sentence to another helpful part of the docs

Comment (jordan-pieces) [reply docs/product-highlights-and-benefits/sharing-to-and-from-github-gists.mdx:18]: this also could be betters as "After signing in and ensuring..." and then starting the steps to share pieces as github gists

File: docs/product-highlights-and-benefits/sharing-to-and-from-github-gists.mdx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+description: GitHub Gists are a great way to share code with a group of people for public visibility, and integrate your snippets with your GitHub repositories. Pieces makes it easy to update your Gist repository by enabling you to share saved code snippets within Pieces as Gists, right from the desktop app.
+---
+
+# Import Gists from GitHub
+
+<Image zoom src="https://storage.googleapis.com/pieces_static_resources/pfd_wiki/IMPORT_GIST.gif" />
+
+Before proceeding to work with Gists from GitHub in Pieces, make sure that you are signed in. _If you are signed in to both Google and GitHub, be sure that the **primary emails on both accounts match**._
+
+## Sharing Pieces snippets as GitHub Gists
```
Comment (jordan-pieces) [inline docs/product-highlights-and-benefits/sharing-to-and-from-github-gists.mdx:12]: i vote Sharing Pieces as Github Gists or Sharing Snippets as Github Gists - both words together feels forced a little but im bad at grammar

File: docs/product-highlights-and-benefits/sharing-to-and-from-github-gists.mdx [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+4. (Optional) Create a title and a description. You can create a title and a description from here, or you can skip this step and a generated title will be created for you.
+5. Press "Generate," and you'll get the link to the Gist, where it can be found on your list of public gists on your GitHub account.
+
+#### Bundled Metadata Preview
+Every Gist has a _Markdown Preview_ built using our <a target="_blank" href="{{ links.internal.context_awareness_engine.origin_details }}">Context Awareness Engine and Origin Details</a> which includes relevant and related information to the snippet's context.
+
+
+## How to Import Gists to Pieces Desktop
+
+
```
Comment (jordan-pieces) [inline docs/product-highlights-and-benefits/sharing-to-and-from-github-gists.mdx:31]: extra space here

File: docs/product-highlights-and-benefits/sharing-to-and-from-github-gists.mdx [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+To import the Gists and upgrade them:
+1. Navigate to 'Add Snippets' > 'Import Gists'
+2. The Snippet Discovery window will open, as we use our Snippet Discovery feature to [enrich your snippets](/features/auto-enrichment) as you import them from GitHub. This way, your Gists will maintain their context.
```
Comment (jordan-pieces) [inline docs/product-highlights-and-benefits/sharing-to-and-from-github-gists.mdx:34]: potentially link to in-project-snippet-discovery?

File: docs/product-highlights-and-benefits/sharing-to-and-from-github-gists.mdx [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+3. Once you press "Select Gists," in a similar way as [Snippet Discovery](/product-highlights-and-benefits/in-project-snippet-discovery), you will be presented with a list of Snippets that have been discovered from your GitHub account.
+
+    If you're not seeing your Gists, make sure they are set to public. Otherwise, they will not be detected.
```
Comment (jordan-pieces) [inline docs/product-highlights-and-benefits/sharing-to-and-from-github-gists.mdx:37]: im actually not sure if this is true, were you able to confirm or deny this ?

File: docs/product-highlights-and-benefits/sharing-to-and-from-github-gists.mdx [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+4. Select the Gists that you want to save by pressing the checkbox to select each one individually or by pressing "save all" in the top right.
+5. You will be brought back to your default view where you can see all of the snippets that you just discovered from your Gists.
+
+### Gist Metadata
+Each code snippet that you save as a Gist is packed full of metadata and information about the repository you saved it from. Traditionally, when you save a Gist, its origin is lost unless you take extra steps to click through dozens of pull requests and commits.
```
Comment (jordan-pieces) [inline docs/product-highlights-and-benefits/sharing-to-and-from-github-gists.mdx:42]: could link to context awareness engine somewhere in here

**Review by jordan-pieces — COMMENTED** [review]
looking really good just little things and considerations but make those changes and we should be good to go

File: docs/product-highlights-and-benefits/sharing-to-and-from-github-gists.mdx [hunk 9]
```diff
@@ -0,0 +1,53 @@
+---
+title: Sharing To & From GitHub Gists
```
Comment (jordan-pieces) [inline docs/product-highlights-and-benefits/sharing-to-and-from-github-gists.mdx:2]: also the old page will need a redirect setup on it as well for a bit

**Review by jordan-pieces — CHANGES_REQUESTED** [review]

File: docs/product-highlights-and-benefits/sharing-to-and-from-github-gists.mdx (hunk 7 above)
Comment (wes-at-pieces) [reply docs/product-highlights-and-benefits/sharing-to-and-from-github-gists.mdx:36]: From what I can tell yes this is accurate, what exactly are you suspicious of?

Comment (jordan-pieces) [reply docs/product-highlights-and-benefits/sharing-to-and-from-github-gists.mdx:36]: just wanted to make sure that you have to have your gists set to public in order for pieces to discover them, if you can just try setting gists to private and then testing if you can discover them via Pieces that would be great

**Review by jordan-pieces — APPROVED** [review]
nice looking great @wes-at-pieces !!

**Pull request merged**